### PR TITLE
Uniform

### LINKED ISSUE #266: Create uniform picker
Create a page where faculty head can choose uniform that everyone should wear. This data is stored in the database. Any milgroup commander can fetch this data via tgbot.

### PR DESCRIPTION
**Pull request by gakhromov**
This pull request should close #266.

### CODE REVIEW

**Comment by gakhromov** [comment]
For now, back-end is done, front-end has been started. Tgbot has not yet been started.

**Comment by TmLev** [comment]
Do you want me to implement `tgbot` part?

**Comment by gakhromov** [comment]
> Do you want me to implement `tgbot` part?

Sure, I'd love so!

File: front-end/src/components/Personnel/UniformPicker/UniformPicker.vue [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+          getError("информации о форме одежды", err.response.status);
+        });
+    },
+    handleClick(obj) {
+      this.fetchUniform(obj.label);
+    },
+    cycleThroughHeaddresses() {
+      var headdress_index = this.headdresses.indexOf(this.uniform.headdress);
+      headdress_index = (headdress_index + 1) % this.headdresses.length;
+      this.uniform.headdress = this.headdresses[headdress_index];
```
Comment (TmLev) [inline front-end/src/components/Personnel/UniformPicker/UniformPicker.vue:95]:
```js
      const nextIndex = this.headdresses.indexOf(this.uniform.headdress) + 1;
      this.uniform.headdress = this.headdresses[nextIndex % this.headdresses.length];
```

File: front-end/src/components/Personnel/UniformPicker/UniformPicker.vue [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    },
+    cycleThroughOuterwears() {
+      var outerwear_index = this.outerwears.indexOf(this.uniform.outerwear);
+      outerwear_index = (outerwear_index + 1) % this.outerwears.length;
+      this.uniform.outerwear = this.outerwears[outerwear_index];
```
Comment (TmLev) [inline front-end/src/components/Personnel/UniformPicker/UniformPicker.vue:100]:
```js
      const nextIndex = this.outerwears.indexOf(this.uniform.outerwear) + 1;
      this.uniform.outerwear = this.outerwears[nextIndex % this.outerwears.length];
```

File: front-end/src/components/Personnel/UniformPicker/UniformPicker.vue [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        .catch((err) => {
+          patchError("формы одежды", err.response.status);
+        });
+    },
+  },
+  created() {
+    getMilFaculties()
+      .then((response) => {
+        this.milfaculties = response.data;
+        this.fetchUniform(this.milfaculties[0].milfaculty);
```
Comment (TmLev) [inline front-end/src/components/Personnel/UniformPicker/UniformPicker.vue:122]: Why 0?

**Review by TmLev — APPROVED** [review]
New component with visual interface, great!

**Pull request merged**